packages/components/menu/src/menu-item.scss

```diff
@@ -56,34 +67,34 @@ sl-icon[name='chevron-right'] {
   position: fixed;
 
   &:hover + [part='wrapper'] {
-    background: var(--sl-elevation-surface-raised-default-hover);
+    --_bg-opacity: var(--sl-opacity-interactive-plain-hover);
   }
 }
 
 [part='wrapper'] {
   align-items: center;
-  background: var(--sl-elevation-surface-raised-default-idle);
+  background: color-mix(in srgb, var(--_bg-color), var(--_bg-mix-color) calc(100% * var(--_bg-opacity)));
   border-radius: var(--sl-size-borderRadius-default);
   cursor: pointer;
   display: flex;
   gap: var(--sl-space-new-md);
   padding: var(--sl-space-new-sm) var(--sl-space-new-md);
   padding-inline-start: calc(
-    var(--sl-menu-item-indent, 0) * (var(--sl-size-icon-sm) + var(--sl-space-new-md)) + var(--sl-space-new-md)
+    var(--sl-menu-item-indent, 0) * (var(--sl-size-new-icon-sm) + var(--sl-space-new-md)) + var(--sl-space-new-md)
```

Comment:
Please don't use `--sl-space-*` tokens, but use `--sl-size-<number>` instead. The former will eventually be removed/not exported to CSS anymore.

Reply:
This applies to all `--sl-space-*` tokens used in this PR :)